cocos/ui/UIButton.cpp

```diff
@@ -788,12 +789,17 @@ void Button::setTitleFontSize(float size)
     {
         _titleRenderer->setSystemFontSize(size);
     }
-    else
+    else if (_type == FontType::TTF)
     {
         TTFConfig config = _titleRenderer->getTTFConfig();
         config.fontSize = size;
         _titleRenderer->setTTFConfig(config);
     }
+    else
+    {
+        // BMFont can't set its font size
+        size = _fontSize;
```

Comment:
why change the argument here? If BMFont can't set it's font size, it should just omit it.